Code/Editor/MainWindow.cpp

```diff
@@ -845,78 +852,183 @@ void MainWindow::InitActions()
         .SetShortcut(tr("Z"))
         .SetToolTip(tr("Center on Selection (Z)"))
         .Connect(&QAction::triggered, this, &MainWindow::OnGotoSelected);
+
+    auto goToViewBookmark = [](int index)
+    {
+        if (AzToolsFramework::ViewBookmarkLoaderInterface* bookmarkLoader = AZ::Interface<ViewBookmarkLoaderInterface>::Get())
+        {
+            const AZStd::optional<AzToolsFramework::ViewBookmark> bookmark =
+                bookmarkLoader->GetBookmarkAtIndex(index, ViewBookmarkLoaderInterface::StorageMode::Local);
+
+            if (bookmark.has_value())
+            {
+                auto viewportContextManager = AZ::Interface<AZ::RPI::ViewportContextRequestsInterface>::Get();
+                if (!viewportContextManager)
+                {
+                    return false;
+                }
+
+                auto viewportContext = viewportContextManager->GetDefaultViewportContext();
+                if (!viewportContext)
+                {
+                    return false;
+                }
+
+                //Check the bookmark we want to load is not exactly 0;
```

Comment:
uber nit: comments luckily don't need a ';' 😛 (and can we add a space after the //)

Reply:
Copy-paste issue